11ty/index.njk

```diff
@@ -3,22 +3,22 @@ layout: layouts/index.njk
 ---
 
 <section id="introduction" class="auto-grid">
-      <h1 class="title__thicc" id="title">Self-Defined</h1>
-      <p>A modern dictionary about us.<br>We define our words, but they don't define us.</p>
-      <div>
-        <p class="summary">
-          Self-Defined seeks to provide more inclusive, holistic, and fluid definitions to reflect the diverse perspectives of the modern world.
-        </p>
-        <p>
-          With the foundation of vocabulary, we can begin to understand lived experiences of people different than us. Words can provide us with a sense of identify and allow us to find kinship through common experiences.
-        </p>
-      </div>
-      <div class="box">
-        <h2 class="subtitle">Ways to help</h2>
-        <ol class="help">
-          <li>Submit words and definitions or contribute to our code base through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull requests</a> or <a href="https://github.com/tatianamac/selfdefined/issues" rel="noreferral">issues</a>. Start with our <a href="https://github.com/tatianamac/selfdefined/blob/master/CONTRIBUTING.md" rel="noreferral">contributing guidelines</a> and <a href="/documentation/">documentation</a>.</li>
-          <li>Sponsor this work through <a href="https://opencollective.com/selfdefined">Open Collective</a> or <a href="https://github.com/sponsors/tatianamac">GitHub Sponsors</a>.</li>
-          <li>Follow us on Twitter <a href="https://www.twitter.com/selfdefinedapp">@SelfDefinedApp</a>.</li>
-        </ol>
-      </div>
-      </section>
+  <h1 class="title title--thicc" id="title">Self-Defined</h1>
+  <p>A modern dictionary about us.<br>We define our words, but they don't define us.</p>
+  <div>
+    <p class="summary">
+      Self-Defined seeks to provide more inclusive, holistic, and fluid definitions to reflect the diverse perspectives of the modern world.
+    </p>
+    <p>
+      With the foundation of vocabulary, we can begin to understand lived experiences of people different than us. Words can provide us with a sense of identify and allow us to find kinship through common experiences.
+    </p>
+  </div>
+  <div class="box">
+    <h2 class="subtitle">Ways to help</h2>
+    <ol class="box__list">
```

Comment:
This list is a structure that can stand on its own. Maybe we shouldn’t use an element class here, but a new block. Agreed, though, that `help` wasn’t the most useful name as it is too tightly coupled to the content.